lib/tizen_plugins.dart

```diff
@@ -94,7 +103,7 @@ mixin DartPluginRegistry on FlutterCommand {
   Future<FlutterCommandResult> verifyThenRunCommand(String? commandPath) async {
     final FlutterProject project = FlutterProject.current();
     final TizenProject tizenProject = TizenProject.fromFlutter(project);
-    if (_usesTargetOption && tizenProject.existsSync()) {
+    if (_usesTargetOption && tizenProject.manifestFile.existsSync()) {
```

Comment:
I deleted the part of checking `tizen-manifest.xml` in `TizenProject.existsSync()` method to return `true` in case of the plugin project. There is no `tizen-manifest.xml` in `tizen` directory of the plugin project. 
So I changed this line to prevent copying the ephemeral directory to `tizen` directory of the plugin project.

Reply:
I fixed this line for the code readability:
```dart
if (_usesTargetOption && tizenProject.isApplication) {
```